rockspec/snowflake-2.1-0.rockspec

```diff
@@ -0,0 +1,28 @@
+package = "snowflake"
```

Comment:
The new package requires a "api7-" prefix, like this one: https://github.com/api7/lua-resty-http/blob/master/api7-lua-resty-http-0.1.0-0.rockspec

Reply:
Does the c library need to add a prefix? If someone uses snowflake.so, there may still be conflicts

Reply:
I think it is fine, nobody will use two snowflake libraries.

Reply:
I think we can change the default value of `SNOWFLAKE_EPOC` back to the original value, so that it can still be compatible with the original library.
> 1609459200000                      =>    1413817200000
> 2021-01-01T00:00:00.000Z    =>     2014-10-20T15:00:00.000Z